### PR TITLE
Add crypto functions to ssmgr.h

### PR DESCRIPTION
**Pull request by Princess-of-Sleeping**

### CODE REVIEW

File: include/psp2kern/kernel/ssmgr.h [hunk 1]
```diff
@@ -42,6 +42,19 @@ int ksceSblAimgrGetConsoleId(SceConsoleId *cid);
 int ksceSblAimgrGetOpenPsId(SceOpenPsId *open_psid);
 int ksceSblAimgrGetPscode(ScePsCode *pscode);
 
+int ksceSblDmac5AesCbcDec(const void *src, void *dst, int size, const void *key, int key_size, void *iv, int mask_enable);
+int ksceSblDmac5AesCbcEnc(const void *src, void *dst, int size, const void *key, int key_size, void *iv, int mask_enable);
+int ksceSblSsMgrAesCtrDecrypt(const void *src, void *dst, int size, const void *key, int key_size, void *iv, int mask_enable);
+
+#define ksceSblDmac5AesCtrDec ksceSblSsMgrAesCtrDecrypt
```
Comment (d3m3vilurr) [inline include/psp2kern/kernel/ssmgr.h:49]: why did you define it?

Comment (Princess-of-Sleeping) [reply include/psp2kern/kernel/ssmgr.h:49]: It's registered in db.yml with ksceSblSsMgrAesCtrDecrypt, but obviously it's ksceSblDmac5AesCtrDec.

Comment (d3m3vilurr) [reply include/psp2kern/kernel/ssmgr.h:49]: hm... okay. first time i thought it's good name than before. but when we try to change it, we have to change every `ksceSblSsMgr*Decrypt/Encrypt`  functions in db.

Comment (Princess-of-Sleeping) [reply include/psp2kern/kernel/ssmgr.h:49]: yes, true.
maybe i'll do some function check.

**Pull request merged**